### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.count_of_ratings = count_of_ratings
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+
+    def serve_cars(self, cars: list) -> float:
```
Comment (Vasyl-Smutok) [inline app/main.py:18]:
```suggestion
    def serve_cars(self, cars: list[???]) -> float:
```

**Review by vsmutok — CHANGES_REQUESTED** [review]
Please use `black` to fix indentation

**Review by vsmutok — CHANGES_REQUESTED** [review]
```
pip install black
black main.py
```

**Review by vsmutok — APPROVED** [review]
Good Job 👍

**Pull request closed without merging**